MandelaK/SendIT-API-V2
# [#162300307]fix bug where admin is duplicated every time the app starts

### What does this PR do?
Fix bug where admin was being duplicated whenever the app starts resulting in an integrity error from psycopg2.

Some tests have also been renamed, but their functionality remains exactly the same

### Description of task to be accomplished
Our app creates an admin each time it is started. I had implemented a function for deleting the admin that was to be called before we created an admin. The problem was sometimes, especially when the server was abruptly disconnected, restarting the app resulted in the admin being duplicated and not deleted, and this always threw an integrity error because the admin email should be unique. I have instead added a query for deleting the admin in the same function that creates the admin. In essence this means that to create an admin, if an existing one is found with the specified email address, they are first deleted and so we cannot have an integrity error. We also now only use one function to both create and delete the admin.

### How can this be manually tested?
If you start the app now, no matter how many times it's restarted, the admin will always be properly deleted before they are recreated again. You can check the development database and you'll notice that they are always the most recent users added whenever the app starts. 

### PT stories
#162300307
